modules/monitoring-granting-users-permission-to-monitor-user-defined-projects.adoc

```diff
@@ -13,21 +13,43 @@ You can also grant developers and other users different permissions:
 * To monitor user-defined projects.
 * To configure the components that monitor user-defined projects.
 * To configure alert routing for user-defined projects.
+* To manage alerts and silences for user-defined projects.
 
-You can grant the permissions by assigning one of the following monitoring roles:
+You can grant the permissions by assigning one of the following monitoring roles and cluster roles:
 
+[IMPORTANT]
+====
+Monitoring cluster roles can be bound with `RoleBinding` to any user project. The monitoring roles can be only bound to the `openshift-user-workload-monitoring` project.
+====
+
+.Monitoring roles
 |===
 |Role name |Description
 
-|`monitoring-rules-view` | Users with this cluster role have read access to `PrometheusRule` custom resources for a user-defined project. They can also view the alerts in the *Developer* perspective of the {product-title} web console.
+|`user-workload-monitoring-config-edit` 
+|Users with this role can edit the `user-workload-monitoring-config` `ConfigMap` object to configure Prometheus, Prometheus Operator, Alertmanager, and Thanos Ruler for user-defined workload monitoring. 
+
+|`monitoring-alertmanager-api-reader` 
+|Users with this role have read access to the user-defined Alertmanager API for all projects, if the user-defined Alertmanager is enabled.
+
+|`monitoring-alertmanager-api-writer` 
+|Users with this role have read and write access to the user-defined Alertmanager API for all projects, if the user-defined Alertmanager is enabled.
+|===
+
+.Monitoring cluster roles
+|===
+|Cluster role name |Description
```

Comment:
(discussion) the other table for core monitoring has a `project` column. Should have it here too?

Reply:
I added a project column to the other one so that it differentiates between the roles and that one cluster role. I could do the same here, but it would be just a lot of repetition I think :frowning_face:  That is why, instead, I decided to add the warning: 	
Monitoring cluster roles can be bound with RoleBinding to any user project. The monitoring roles can be only bound to the openshift-user-workload-monitoring project.

But if you think it would be better to add that column, I can :slightly_smiling_face: maybe that would be clearer :+1:

Reply:
That's fine by me. I just wanted to make sure that we were good with it :)